pkg/resolver/searchComplete_test.go

```diff
@@ -98,7 +97,12 @@ func Test_SearchCompleteWithFilter_Query(t *testing.T) {
 	csRes, nsRes, managedClusters := newUserData()
 	ud := rbac.UserData{CsResources: csRes, NsResources: nsRes, ManagedClusters: managedClusters}
 	searchInput := &model.SearchInput{Filters: []*model.SearchFilter{{Property: "namespace", Values: []*string{&value1, &value2}}, {Property: "cluster", Values: []*string{&cluster}}}, Limit: &limit}
-	resolver, mockPool := newMockSearchComplete(t, searchInput, prop1, &ud)
+	PropTypes := make(map[string]string)
+	PropTypes["kind"] = "string"
+	PropTypes["namespace"] = "string"
+	PropTypes["cluster"] = "string"
```

Comment:
Simpler to declare inline
```suggestion
	PropTypes := make(map[string]string) { "kind":"string", "namespace":"string", "cluster":"string" }
```